pkg/frontend/frontend_test.go

```diff
@@ -16,11 +16,11 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
-	"github.com/sirupsen/logrus/hooks/test"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/Azure/ARO-RP/pkg/api"
+	test_log "github.com/Azure/ARO-RP/test/util/log"
```

Comment:
Pythonista!  Golang convention avoids underscores: https://golang.org/doc/effective_go.html#mixed-caps

Reply:
Oh, huh. I've been unsure about it and copied that from `install_test.go` [1], trying to be consistent. Seems I was consistent in inconsistency!

[1]:
https://github.com/Azure/ARO-RP/blob/7b95461cda89e9f4c6cc9cf1ae004b056d303f5e/pkg/install/install_test.go#L20-L21

Reply:
(Amusingly, I've spent the last 7 years of writing Python being told to use underscores, since the Twisted coding style is mixedCaps, with underscores used only for dispatch functions (e.g. `request_GET`).)

Reply:
Mmm.  Gomock doesn't follow the convention, it transpires.  Let's not make things worse and call this testlog instead.

Reply:
gomock's mockgen does allow us to override it with the `-package` switch, if we wished.